modules/inflowwind/src/Lidar.txt

```diff
@@ -37,6 +37,19 @@ typedef	^	Lidar_ParameterType	ReKi	DeltaP    	-	-	-	"the distance between range
 typedef	^	Lidar_ParameterType	ReKi	DeltaR    	-	-	-	"the FWHM width of the pulse"
 typedef	^	Lidar_ParameterType	ReKi	r_p	-	-	-
 typedef	^	Lidar_ParameterType	LOGICAL	LidRadialVel	-	-	-	"TRUE => return radial component, FALSE => return 'x' direction estimate"	-
+typedef	^ Lidar_ParameterType	ReKi	DisplacementLidarX		-  	-   -	"Displacement of the lidar system from the focal measurement point"	-
```

Comment:
Please fix the column alignment.

Reply:
I've fixed the column alignment throughout the .txt files